scripts/presubmit.sh

```diff
@@ -77,6 +77,14 @@ main() {
     grep -v asn1/ | \
     tr '\n' ' ')"
 
+  # Prevent the creation of proto files with .txt extensions.
+  bad_protos="$(find . -name '*.pb.txt' -o -name '*.proto.txt')"
+  if [[ "${#bad_protos}" -ne 0 ]]; then
```

Comment:
This might be more clearly expressed with an equals operator. I had to go lookup what `${#foo}` does (in this case, it evaluates to the length of the string, but its behaviour varies depending on what type the variable is), whereas I think it's pretty obvious what the semantically-identical equals operator does.

```suggestion
  if [[ "${bad_protos}" != "" ]]; then
```